src/components/Nav.js

```diff
@@ -7,8 +7,8 @@ function Nav() {
         Bookstore CMS
       </h1>
       <nav className="options">
-        <Link to="/"> [BOOKS] </Link>
-        <Link to="Categories"> [CATEGORIES] </Link>
+        <Link to="/"> BOOKS </Link>
+        <Link to="Categories"> CATEGORIES </Link>
```

Comment:
- I would recommend using the `NavLink` instead of the `Link` component here. This will add a default active class which will help in highlighting the active link in the navbar just like in zeplin design.
- I would recommend adding the user icon in the top right corner just like in the zeplin design.

| Your design  | zeplin design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/224485118-8a47fe17-fb01-46c4-94ec-a7befdda38cd.png) | ![image](https://user-images.githubusercontent.com/33268927/224485050-b36a0886-9e22-4fbd-80a5-48b64a12038f.png) |

Reply:
Ok, your recommendation is great, my page looks better